augmentedreality/Examples/QmlArExample/Android/AndroidManifest.xml

```diff
@@ -1,86 +1,64 @@
 <?xml version="1.0"?>
-<manifest package="com.esri.arcgisruntime.qt.qml.ar.example" xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="-- %%INSERT_VERSION_NAME%% --" android:versionCode="-- %%INSERT_VERSION_CODE%% --" android:installLocation="auto">
-    <uses-sdk android:minSdkVersion="26" android:targetSdkVersion="28"/>
```

Comment:
I think we need to keep android:minSdkVersion and android:targetSdkVersion. Are you sure these new changes are ok?